aiida/cmdline/commands/cmd_code.py

```diff
@@ -168,7 +168,36 @@ def code_duplicate(ctx, code, non_interactive, **kwargs):
 @with_dbenv()
 def show(code, verbose):
     """Display detailed information for a code."""
-    click.echo(tabulate.tabulate(code.get_full_text_info(verbose)))
```

Comment:
So, can I ask what is the intended purpose of the commands `show` and `computer_show` modified in this PR? Because I'm not sure I understand why you are removing the general `get_full_text_info` method to implement two relatively similar versions directly into each of those commands.

Reply:
Note that one is `verdi code show` and the other `verdi computer show`. I removed the methods from the classes because they don't really belong there. Especially the `Computer` one was bad since it was baking in the formatting. It was returning a string fully formatted. This is really inflexible as the caller cannot do anything to modify it. The `Code` one is already slightly better since it simply returned a list of tuples. Still I think deciding what information should be displayed is still very situation dependent and so there is no need to have a dedicated method for this. It was no wonder that the methods were also just used in one place, namely the `show` CLI commands. Might as well put that logic there so it has all the freedom it needs to choose what and how to display the information

Reply:
Ah, I see, I didn't noticed these were methods of different classes. Thanks!